test/extensions/filters/http/ext_proc/ordering_test.cc

```diff
@@ -71,7 +71,7 @@ class OrderingTest : public testing::Test {
       (*cb)(proto_config);
     }
     config_.reset(new FilterConfig(proto_config, kMessageTimeout, kMaxMessageTimeoutMs,
-                                   *stats_store_.rootScope(), ""));
+                                   *stats_store_.rootScope(), "", Envoy::Extensions::Filters::Common::Expr::BuilderInstance(Envoy::Extensions::Filters::Common::Expr::createBuilder(nullptr)).builder()));
```

Comment:
This is how this builder is initialized on the upstream PR

```suggestion
                                   *stats_store_.rootScope(), "", std::make_shared<Envoy::Extensions::Filters::Common::Expr::BuilderInstance>(
            Envoy::Extensions::Filters::Common::Expr::createBuilder(nullptr))));
```